src/RowsContainer.js

```diff
@@ -1,14 +1,44 @@
 import React, {PropTypes} from 'react';
-import {ContextMenuLayer} from 'react-contextmenu';
+
+const SimpleRowsContainer = (props) => {
+  return (
+    <div style={{width: props.width, overflow: 'hidden'}}>
+      {props.rows}
+    </div>
+  );
+};
+
+SimpleRowsContainer.propTypes = {
+  width: PropTypes.number,
+  rows: PropTypes.array
+};
 
 class RowsContainer extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {ContextMenuContainer: this.getContextMenuContainer()};
+    this.hasContextMenu = this.hasContextMenu.bind(this);
+    this.renderRowsWithContextMenu = this.renderRowsWithContextMenu.bind(this);
+    this.getContextMenuContainer = this.getContextMenuContainer.bind(this);
+  }
+
+  getContextMenuContainer() {
+    if (!(window.ReactDataGridPlugins && window.ReactDataGridPlugins)) {
+      throw new ('You need to include ReactDataGrid UiPlugins in order to initialise context menu');
```

Comment:
Lets throw a new Error [source](http://stackoverflow.com/questions/9156176/what-is-the-difference-between-throw-new-error-and-throw-someobject-in-javas) instead